### PR TITLE
add popup more details

### PR DESCRIPTION
**Pull request by omarZrayka23**
move salary range, sms language and whatsapp fields under the profession field

add a button that opens a popup for ‘more details’ next to the work address details button, this popup will contain these fields:

transaction count per year

transaction amount per year

risk level

civil status

old reference

title

professionId field should be moved from clientIndividual to clientMaster 

add 2 new fields in the new popup: 
- Extra Income (optional, binds to extraIncome, numberfield, max 12 digits) 
- Extra Income Type (optional, binds to extraIncomeId combobox that fills from RTSET.EI - show reference and name in the dropdown and on selection)

### CODE REVIEW

**Comment by Ghinwa-Arab** [comment]
image

Can we increase the height slightly so that these two fields are fully visible without requiring scrolling

**Comment by Ghinwa-Arab** [comment]
image

1. all save buttons in popups should be disabled in editMode
2. extra income fields should be added in 'more details' popup , don't create third one

File: src/pages/clients-list/forms/ClientTemplateForm.js [hunk 1]
```diff
@@ -1600,164 +1460,212 @@ const ClientTemplateForm = ({ recordId, labels, plantId, maxAccess, allowEdit =
...
+                        })
+                      }
+                    >
+                      {labels.moreDetails}
+                    </Button>
+                    <Button
+                      sx={{ ml: 2 }}
+                      variant='contained'
+                      onClick={() =>
+                        stack({
```
Comment (Ghinwa-Arab) [inline src/pages/clients-list/forms/ClientTemplateForm.js:1530]: remove this button

File: src/pages/clients-list/forms/ExtraIncome.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  useEffect(() => {
+    formik.setValues({
+      extraIncome: clientFormik.values.extraIncome,
+      extraIncomeId: clientFormik.values.extraIncomeId
+    })
+  }, [])
+
+  return (
+    <FormShell form={formik} infoVisible={false} isCleared={false}>
```
Comment (Ghinwa-Arab) [inline src/pages/clients-list/forms/ExtraIncome.js:36]: fields should added to more details button

File: src/repositories/RemittanceRepository.js [hunk 3]
```diff
@@ -153,5 +153,8 @@ export const RemittanceSettingsRepository = {
     get: service + 'getCGP',
     set: service + 'setCGP',
     del: service + 'delCGP'
+  },
+  ExtraIncome: {
```
Comment (Ghinwa-Arab) [inline src/repositories/RemittanceRepository.js:157]: merge from master and check if this endpoint already present.
if it's please remove yours

Comment (omarZrayka23) [reply src/repositories/RemittanceRepository.js:157]: not alrady present

**Pull request merged**